index.js

```diff
@@ -0,0 +1,94 @@
+//Arrays and Cycles
+function check1() {
+  let arr = ["film 1", "film 2", "film 3", "film 4"];
+  for (let elem of arr) {
+    console.log(elem);
+  }
+}
+check1();
+
+function check2() {
+  let arr = ["bmw", "ford", "renault", "audi"];
+  let str = arr.join(",");
+  let arr2 = str.split(",");
+  console.log(arr2);
+}
+check2();
+
+function check3() {
+  let arr = ["ivan", "ura", "vova", "sasha"];
+  let arr2 = [];
+  for (let elem of arr) {
+    elem = "privet" + " " + elem;
+    arr2.push(elem);
+  }
+  console.log(arr2);
+}
+check3();
+
+function check4() {
+  let arr = [1, 6, 7, 8, 3, 4, 5, 6, ""];
+  let arr2 = [];
+  for (let elem of arr) {
+    let boolean = Boolean(elem);
+    arr2.push(boolean);
+    console.log(arr2);
+  }
```

Comment:
Do you know about method `map()`? https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/map